libraries/helpers.rb

```diff
@@ -27,6 +27,10 @@ def default_user_attributes
         Mash.new(home: '/', shell: '/sbin/nologin')
       end
     end
+
+    def series_to_version(series)
+      series.to_s.chars.map(&:to_i).join('.') + '.*'
```

Comment:
this will break when they hit `10.0` or `4.10`. I don't have a good answer how to solve this off the top of my head other than requiring specifying the version.